neurolang/probabilistic/tests/test_marg_query_resolution.py

```diff
@@ -125,7 +125,7 @@ def test_marg_query_two_vars_conditioning(solver):
 
 
 def test_marg_query_conjunctive_conditioned_and_conditioning(solver):
-    if solver != dichotomy_theorem_based_solver:
+    if solver != small_dichotomy_theorem_based_solver:
```

Comment:
This shouldn't be skipped an exception should be returned and verified. There's examples of this in other cases